charts/nublado/values.yaml

```diff
@@ -242,3 +246,62 @@ jupyterhub_config: |
   # External proxy
   c.ConfigurableHTTPProxy.should_start = False
   c.ConfigurableHTTPProxy.api_url = lc.proxy_api_url
+
+base_passwd_file: |
+  root:x:0:0:root:/root:/bin/bash
+  bin:x:1:1:bin:/bin:/sbin/nologin
+  daemon:x:2:2:daemon:/sbin:/sbin/nologin
+  adm:x:3:4:adm:/var/adm:/sbin/nologin
+  lp:x:4:7:lp:/var/spool/lpd:/sbin/nologin
+  sync:x:5:0:sync:/sbin:/bin/sync
+  shutdown:x:6:0:shutdown:/sbin:/sbin/shutdown
+  halt:x:7:0:halt:/sbin:/sbin/halt
+  mail:x:8:12:mail:/var/spool/mail:/sbin/nologin
+  operator:x:11:0:operator:/root:/sbin/nologin
+  games:x:12:100:games:/usr/games:/sbin/nologin
+  ftp:x:14:50:FTP User:/var/ftp:/sbin/nologin
+  nobody:x:99:99:Nobody:/:/sbin/nologin
+  systemd-network:x:192:192:systemd Network Management:/:/sbin/nologin
+  dbus:x:81:81:System message bus:/:/sbin/nologin
+  lsst_lcl:x:1000:1000::/home/lsst_lcl:/bin/bash
+  tss:x:59:59:Account used by the trousers package to sandbox the tcsd daemon:/dev/null:/sbin/nologin
+  provisionator:x:769:769:Lab provisioning user:/home/provisionator:/bin/bash
```

Comment:
Where is this file coming from? It feels weird to hard-code all of this in the chart, since the chart probably doesn't care whether, e.g., the `games` user exists. It also means that to instantiate users or (probably more relevantly) groups, an overriding `values.yaml` file has to duplicate all of this data, which proliferates copies of it.

Shouldn't we instead define only the additions and otherwise preserve the file in the underlying Docker image?